### PR TITLE
Visual polish and some style tweaks

### PR DESCRIPTION
**Pull request by rossmoody**
Visual polish and some structural updates. Code is likely not optimal but for the most part it's just swapped imagery and tiny spacing updates.

### CODE REVIEW

File: stories/features/welcome/page/index.tsx [hunk 1]
```diff
@@ -133,7 +133,7 @@ class WelcomePage extends React.PureComponent<{}, WelcomePageState> {
           </Select>
           <Button
             level='primary'
-            type={this.state.fakeChangedDefaultTheme ? 'default' : 'accent'}
+            type={this.state.fakeChangedDefaultTheme ? 'accent' : 'accent'}
```
Comment (rossmoody) [inline stories/features/welcome/page/index.tsx]: Could be a possibility to introduce a "success" button style of some sort that incorporates green. We have "warn" for red.

File: stories/assets/img/welcomebg.svg [hunk 2]
```diff
@@ -1,16 +1,16 @@
-<svg width='2880' height='600' xmlns='http://www.w3.org/2000/svg'>
+<svg width='2880' height='1000' xmlns='http://www.w3.org/2000/svg'>
```
Comment (cezaraugusto) [inline stories/assets/img/welcomebg.svg]: is there a need for this file to be so large? I noticed a little junk when animations happen, reducing file dimentions could help.

File: stories/features/welcome/page/index.tsx (hunk 1 above)
Comment (cezaraugusto) [reply stories/features/welcome/page/index.tsx]: I think it's just a matter of setting the right color. we can then open a new issue and address in a follow-up

File: src/components/buttonsIndicators/button/style.ts [hunk 3]
```diff
@@ -84,7 +84,7 @@ const StyledButton = styled<Props, 'button'>('button')`
   border-radius: ${largeMediumSmall('24px', '20px', '16px', '28px')};
   width: ${p => p.size === 'call-to-action' ? '100%' : 'auto'};
   min-width: ${largeMediumSmall('116px', '104px', '88px', '235px')};
-  padding: ${largeMediumSmall('14px 15px', '11px 10px', '7px 10px', '19px 15px')};
+  padding: ${largeMediumSmall('14px 18px', '11px 12px', '7px 12px', '19px 15px')};
```
Comment (cezaraugusto) [inline src/components/buttonsIndicators/button/style.ts]: please keep in mind that changes in here would affect all buttons across our app, is this intentional?

File: src/components/formControls/select/style.ts [hunk 4]
```diff
@@ -52,7 +53,7 @@ export const StyledWrapper = styled<StyleProps, 'div'>('div')`
 export const StyledSelectWrapper = styled<StyleProps, 'div'>('div')`
   position: relative;
   outline: 0;
-  font-family: Poppins, sans-serif;
+  font-family: Muli, sans-serif;
```
Comment (cezaraugusto) [inline src/components/formControls/select/style.ts]: ditto

**Pull request merged**